server/index-node/src/service.rs

```diff
@@ -120,7 +120,7 @@ where
         let validated = ValidatedRequest::new(body, &req_parts.headers)?;
         let query = validated.query;
 
-        let query = match PreparedQuery::new(&self.logger, schema, None, query, None, 100) {
+        let query = match PreparedQuery::new(&self.logger, schema, None, query, None, 100, None) {
```

Comment:
I'd prefer to use a fake metrics collector like there is for tests, than have this be an option.

Reply:
Done